src/Network/HTTPClient/Capability/ICanRequestPerHttp.php

```diff
@@ -19,14 +19,14 @@
  *
  */
 
-namespace Friendica\Network;
+namespace Friendica\Network\HTTPClient\Capability;
 
 use GuzzleHttp\Exception\TransferException;
 
 /**
  * Interface for calling HTTP requests and returning their responses
  */
-interface IHTTPClient
+interface ICanRequestPerHttp
```

Comment:
"per"? It sounds odd to my American English-trained ear. How about one of these alternatives:
```suggestion
interface ICanPerformHttpRequests
```
```suggestion
interface ICanIssueHttpRequests
```
```suggestion
interface ICanSendHttpRequests
```
I
I'm partial to the latter, we already use the verb `send` in the phpDoc.

Reply:
A little bit too denglish :D